src/py42/_internal/clients/archive.py

```diff
@@ -1,40 +1,86 @@
 import json
 
+from py42 import settings
 from py42.clients import BaseClient
 from py42.clients.util import get_all_pages
 
 
 class ArchiveClient(BaseClient):
     def get_data_key_token(self, device_guid):
-        uri = u"/api/DataKeyToken"
-        data = {u"computerGuid": device_guid}
+        uri = "/api/DataKeyToken"
```

Comment:
Why were these `u`'s deleted?

Reply:
It looks like they were all removed when I ran `tox -e style`. I'll watch out for that in the future.

Reply:
Interesting, I hadn't seen that happen before

Reply:
Looks like it's only happening for `src/py42/_internal/clients/archive.py`. The other files are untouched.